actionview/lib/action_view/template/render_in_object.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module ActionView # :nodoc:
+  class Template # :nodoc:
```

Comment:
Only the new class should be no-doc'd.

```suggestion
  class Template
```